### PR TITLE
[Relay]Allow every runtime module to handle constants

### PR DESCRIPTION
**Pull request by comaniac**
Per discussion in #5884, this PR makes the following improvements:
* Allow any runtime with `get_symbol` and `get_const_vars` to use the unified mechanism of constant processing.
* We will have a JSON runtime PR later this week to demonstrate how a customized runtime works with this mechanism.

cc @zhiics @mbaret

### CODE REVIEW

File: src/target/source/source_module.cc [hunk 1]
```diff
@@ -119,7 +122,6 @@ class CSourceModuleNode : public runtime::ModuleNode {
       return PackedFunc(
           [sptr_to_self, this](TVMArgs args, TVMRetValue* rv) { *rv = this->const_vars_; });
     } else {
-      LOG(FATAL) << "Unknown packed function: " << name;
```
Comment (mbaret) [inline src/target/source/source_module.cc:122]: Is this related?

Comment (comaniac) [reply src/target/source/source_module.cc:122]: Not really but just a quick fix. I could remove this line and send it along with the JSON runtime if you think that's better.

Comment (mbaret) [reply src/target/source/source_module.cc:122]: I think that would be neater. Even a 1 line PR would be fine.

Comment (zhiics) [reply src/target/source/source_module.cc:122]: This would be needed. Your runtime could get unnecessarily stuck here for other queries.

Comment (mbaret) [reply src/target/source/source_module.cc:122]: If it's a necessary but unrelated change, I think a separate PR is most appropriate.

Comment (comaniac) [reply src/target/source/source_module.cc]: done

**Comment by mbaret** [comment]
Given some modules don't use this mechanism, should we find a way to suppress this warning?
https://github.com/apache/incubator-tvm/blob/b94e8b7290c5ced98728e730634ec73727c53c51/src/runtime/graph/graph_runtime.cc#L93
I'm noticing that I'm getting spammed with it now.

**Comment by zhiics** [comment]
@mbaret this should be handled later by changing the way to initialize graph runtime in different PRs.

**Comment by zhiics** [comment]
thanks @comaniac @mbaret @junrushao1994
